### PR TITLE
comdb2_tunables: Fix a crash during self-join (ported from 7.0)

### PR DESCRIPTION
**Pull request by nirbhayc**
Signed-off-by: Nirbhay Choubey <nchoubey@bloomberg.net>

### CODE REVIEW

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>3/401 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>simple_timepart_reptimeout_generated<br>simple_timepart<br>datetime</p><p><a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/68eb5970/detail.txt'>Detail</a> | <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/68eb5970/logs.tar.gz'>Log</a> | <a href='http://ec2-34-202-216-70.compute-1.amazonaws.com/restricted/index.php?id=2748'>Resubmit</a></p>

File: sqlite/ext/comdb2/tunables.c [hunk 1]
```diff
@@ -84,21 +88,29 @@ static int systblTunablesDisconnect(sqlite3_vtab *pVtab)
     return SQLITE_OK;
 }
 
+static void jumpToNextVisibleTunable(systbl_tunables_cursor *cur)
```
Comment (akshatsikarwar) [inline sqlite/ext/comdb2/tunables.c:91]: This works if the last tunable is hidden or we access past the end?

Comment (nirbhayc) [reply sqlite/ext/comdb2/tunables.c:91]: That's checked under systblTunablesNext(). While verifying this, I found another issue though.

**Review by roborivers — CHANGES_REQUESTED** [review]
<p><a href='https://bloomberg.github.io/comdb2/contrib.html#code-style'>Coding style</a> check: Success &#10003;.<br>Smoke testing: Success &#10003;.<br>Cbuild submission: Success &#10003;.<br>Regression testing: <b>6/401 tests failed</b> &#x26A0;.</p><p>The first 10 failing tests are: <br>simple_timepart_reptimeout_generated<br>sc_timepart_logicalsc_generated<br>sc_timepart<br>simple_timepart<br>datetime<br>sc_swapfields_logicalsc_generated</p><p><a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/ad796cff/detail.txt'>Detail</a> | <a href='http://comdb2.s3-website-us-east-1.amazonaws.com/tests/ad796cff/logs.tar.gz'>Log</a> | <a href='http://ec2-34-202-216-70.compute-1.amazonaws.com/restricted/index.php?id=2748'>Resubmit</a></p>

**Review by akshatsikarwar — APPROVED** [review]

**Review by akshatsikarwar — APPROVED** [review]
o hi

**Comment by nirbhayc** [comment]
o hi thx

**Pull request merged**